packages/media/src/webrtc/VegaRtcManager/index.ts

```diff
@@ -782,17 +782,19 @@ export default class VegaRtcManager implements RtcManager {
             try {
                 const currentPaused = this._webcamPaused;
 
+                const mediaSettings = getMediaSettings("video", false, this._features);
                 const producer = await this._sendTransport.produce({
                     track: this._webcamTrack,
                     disableTrackOnPause: false,
                     stopTracks: false,
-                    ...getMediaSettings("video", false, this._features),
+                    ...mediaSettings,
                     appData: {
                         streamId: OUTBOUND_CAM_OUTBOUND_STREAM_ID,
                         sourceClientId: this._selfId,
                         screenShare: false,
                         source: "webcam",
                         paused: currentPaused,
+                        numSimulcastLayers: mediaSettings?.encodings?.length || 0,
```

Comment:
Is this needed for setMaxSpatialLayer to not fail - or just to get the -1?
Will it work for SVC? (do we then specify multiple encodings?)

Reply:
what happens if you user number.MAX_VALUE?

Reply:
Yes, that is easier and seems to work.